enterprise/server/util/container/credentials/credentials.go

```diff
@@ -0,0 +1,112 @@
+package credentials
```

Comment:
similar comment here - wdyt about renaming this to `oci`, `oci_image` `ociregistry`, or `imageregistry`? And maybe move it directly under `util`, since the util dir currently has a flat structure.

So `oci.Registry` / `oci.Credentials` instead of `credentials.Registry` / `credentials.Credentials`.

Reply:
good call, done